symfony/src/Entity/PatientStatus.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\PatientStatusRepository")
+ */
+class PatientStatus
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $participant_id;
```

Comment:
Applies here and in all of the entities. Property names should be camel-cased. Doctrine will automatically use the corresponding underscore-separated column name.

I just looked it up and realized that PSR-2 purposely doesn't include this as a SHOULD/MUST, but it is the default for how auto-generated entities are created and is the standard that we currently use in our other classes.